### ISSUE TITLE
Error when using compass --watch

### ISSUE THREAD

**Issue by joneff** [Compass/compass#19]
```
NoMethodError on line 44 of /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/lib/compass/commands/watch_project.rb: undefined method `>' for nil:NilClass
  /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/lib/compass/commands/watch_project.rb:25:in `perform'
  /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/lib/compass/commands/watch_project.rb:17:in `loop'
  /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/lib/compass/commands/watch_project.rb:17:in `perform'
  /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/lib/compass/commands/base.rb:15:in `execute'
  /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/lib/compass/commands/project_base.rb:19:in `execute'
  /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/lib/compass/exec.rb:197:in `do_command'
  /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/lib/compass/exec.rb:57:in `perform!'
  /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/lib/compass/exec.rb:40:in `run!'
  /home/joneff/.gem/ruby/1.8/gems/chriseppstein-compass-0.6.15/bin/compass:8
  /home/joneff/.gem/ruby/1.8/bin/compass:19:in `load'
  /home/joneff/.gem/ruby/1.8/bin/compass:19
```

**Comment by joneff** [comment]
Only happens if project is not initialized

**Comment by chriseppstein** [comment]
fixed in 0.8
